<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pBtn_Move">
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_InsTransistor">
        <property name="text">
         <string>Транзистор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_InsResistor">
        <property name="text">
         <string>Резистор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_InsCondensator">
        <property name="text">
         <string>Конденсатор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_InsDiode">
        <property name="text">
         <string>Диод</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_InsLine">
        <property name="text">
         <string>Линия</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>74</width>
          <height>118</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="MyGV" name="graphicsView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menu_Insert">
    <property name="title">
     <string>&amp;Вставить</string>
    </property>
    <addaction name="action_Add_Transistor"/>
    <addaction name="action_Add_Diode"/>
    <addaction name="action_Add_Resistor"/>
    <addaction name="action_Add_Capacitor"/>
   </widget>
   <widget class="QMenu" name="menu_DB_Elements">
    <property name="title">
     <string>&amp;Элементная база</string>
    </property>
    <addaction name="action_T_DB"/>
    <addaction name="action_R_DB"/>
    <addaction name="action_C_DB"/>
    <addaction name="action_D_DB"/>
   </widget>
   <widget class="QMenu" name="menu_Act">
    <property name="title">
     <string>&amp;Действия</string>
    </property>
    <addaction name="action_Delete"/>
    <addaction name="Rotate_clockwise"/>
    <addaction name="Rotate_counterclockwise"/>
    <addaction name="Scale_plus"/>
    <addaction name="Scale_minus"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Insert"/>
   <addaction name="menu_DB_Elements"/>
   <addaction name="menu_Act"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_DBElem">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_Act">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Открыть</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Сохранить</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Закрыть</string>
   </property>
  </action>
  <action name="action_Add_Transistor">
   <property name="text">
    <string>&amp;Транзистор</string>
   </property>
  </action>
  <action name="action_Add_Diode">
   <property name="text">
    <string>&amp;Диод</string>
   </property>
  </action>
  <action name="action_Add_Resistor">
   <property name="text">
    <string>&amp;Резистор</string>
   </property>
  </action>
  <action name="action_Add_Capacitor">
   <property name="text">
    <string>&amp;Конденсатор</string>
   </property>
  </action>
  <action name="action_T_DB">
   <property name="text">
    <string>&amp;База транзисторов</string>
   </property>
  </action>
  <action name="action_R_DB">
   <property name="text">
    <string>&amp;База резисторов</string>
   </property>
  </action>
  <action name="action_DB_Transistor">
   <property name="text">
    <string>&amp;Транзистор</string>
   </property>
  </action>
  <action name="action_DB_Resistor">
   <property name="text">
    <string>&amp;Резистор</string>
   </property>
  </action>
  <action name="action_DB_Diode">
   <property name="text">
    <string>&amp;Диод</string>
   </property>
  </action>
  <action name="action_DB_Capacitor">
   <property name="text">
    <string>&amp;Конденсатор</string>
   </property>
  </action>
  <action name="action_C_DB">
   <property name="text">
    <string>&amp;База конденсаторов</string>
   </property>
  </action>
  <action name="action_D_DB">
   <property name="text">
    <string>&amp;База диодов</string>
   </property>
  </action>
  <action name="action_90">
   <property name="text">
    <string>&amp;Повернуть на +90</string>
   </property>
  </action>
  <action name="action_91">
   <property name="text">
    <string>&amp;Повернуть на -90</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Удалить элемент</string>
   </property>
  </action>
  <action name="Rotate_clockwise">
   <property name="text">
    <string>&amp;Повернуть по часовой</string>
   </property>
  </action>
  <action name="Rotate_counterclockwise">
   <property name="text">
    <string>&amp;Повернуть против часовой</string>
   </property>
  </action>
  <action name="Scale_plus">
   <property name="text">
    <string>$Масштаб+</string>
   </property>
  </action>
  <action name="Scale_minus">
   <property name="text">
    <string>$Масштаб-</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>$Сохранить как...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGV</class>
   <extends>QGraphicsView</extends>
   <header location="global">mygv.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
